qiskit/providers/ibmq/managed/ibmqjobmanager.py

```diff
@@ -223,10 +234,10 @@ def retrieve_job_set(
         """Retrieve a previously submitted job set.
 
         Args:
-            job_set_id: ID of the job set.
+            job_set_id: Job set ID.
             provider: Provider used for this job set.
-            refresh: If True, re-query the API for the job set.
-                Otherwise return the cached value. Default: False.
+            refresh: If True, re-query the server for the job set information.
```

Comment:
```suggestion
            refresh: If ``True``, re-query the server for the job set information.
```

Reply:
I debated about this, but the Google Style Example does not surround `True` and `False` with backticks: 
```
    Returns:
        bool: The return value. True for success, False otherwise.
```